shotgun_api3/shotgun.py

```diff
@@ -774,9 +774,6 @@ def info(self):
         >>> sg.info()
         {'full_version': [8, 2, 1, 0], 'version': [8, 2, 1], 'user_authentication_method': 'default', ...}
 
-        Tokens and values
```

Comment:
This was never rendered out to html and generated warnings. It made sense to remove as the doc is prefectly readable without it.